### DISCUSSION TITLE
Can you disable quick note icons?

### DISCUSSION THREAD

**Discussion by MTRG15** [FlorianWoelki/obsidian-icon-folder#296] [Q&A] [answered]
The use of the character ":" automatically displays a menu with the icon selection, this is great, but I would like to disable that function.

I use Obsidian's Flashcard plugin for Anki, and it uses "::" and ":::" which triggers the menu at inconvenient times

Is there a way to disable this feature? Or change the operator for a different symbol that doesn't cause this friction?

**Comment by FlorianWoelki** [comment, marked as answer]
Changing the operator for this might be a good addition to this feature. I've created an issue #297 for that.

**Reply by MTRG15** [reply]
Thanks, I hesitated from posting this as an issue since these two plugins don't really conflict with each other, but the awkwardness can be an issue too
